Getting-Started.md

```diff
@@ -398,6 +482,65 @@ starting/(re)creating/building/attaching containers for a service.
 	```
     And restart the system.
 
+
+#### SLL error [&uarr;](#ssl-)
+
+![ssl_error](https://user-images.githubusercontent.com/53489999/77805355-aea32c00-70a7-11ea-9222-5073851bdc60.png)
+
+* Solution:
+
+  * Change 'https' to 'http' in the setup.cfg file
+
+  ```
+  [es_collection]
+  # arthur = true
+  # arthur_url = http://127.0.0.1:8080
+  # redis_url = redis://localhost/8
+  url = http://ocalhost:9200
+
+  [es_enrichment]
+  url = http://localhost:9200
+  ```
+
+
+#### cloc installation [&uarr;](#cloc-installation-)
+
+![cloc_error](https://user-images.githubusercontent.com/53489999/77805464-f45ff480-70a7-11ea-90ef-04e1c588f2fd.png)
+
+* Solution:
+
+  Execute the following -
+
+  [Refer](https://github.com/chaoss/grimoirelab-graal#how-to-installcreate-the-executables)
+
+  ```
+  sudo apt-get install cloc
+  ```
+
+#### git 'latest-item' error [&uarr;](#latest-item-)
```

Comment:
This troubleshooting is already included at https://github.com/chaoss/grimoirelab-sirmordred/blob/fd41c53ff40fbf86922067d8f856f42a1f29331e/Getting-Started.md#empty-index-, thus it can be removed.

Reply:
Sorry, missed that. Will remove it. Thanks!